holoviews/plotting/bokeh/callbacks.py

```diff
@@ -924,6 +925,14 @@ def _process_msg(self, msg):
                 values = new_values
             elif any(isinstance(v, (int, float)) for v in values):
                 values = [np.nan if v is None else v for v in values]
+            elif isinstance(values, list) and len(values) == 4 and isinstance(values[3], list):
```

Comment:
Don't think this condition is quite sufficient as this could in theory occur when receiving polygon or path data. Probably need one more condition like `values[2] in ('big', 'little')`